### PR TITLE
Definition of "sprint"

### PR DESCRIPTION
**Pull request by krystenkoehn**
***Maybe the second paragraph is unnecessary?

« A sprint, in conventional terms, is ‘an act or short spell of running
at full speed over a short distance’. For New Atoms, a sprint,
similarly, indicates a short duration of steady working that is fixed
in advance (at this point, around two days).

Before a sprint, the team votes on which Achievables will be put into
‘Next’ in the Achievables board. During a sprint, each team member
works consistently on Achievables (also called "cards") from the Next
column of the Interface and reviews the Achievables fulfilled by other
team members. »

### CODE REVIEW

**Comment by SPrinss** [comment]
Could you add some hyperlinks to the other terms used in the explanation? 
For example 'achievables' 
:+1:

**Comment by krystenkoehn** [comment]
I shall! What do you think about the second paragraph? Is it unnecessary?

**Comment by SPrinss** [comment]
Perhaps you could give a link to the board guide and explain that a sprint is from voting in achievables to demoing them.

**Comment by krystenkoehn** [comment]
Good idea. Consider it done!

File: glossary/sprint.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+A sprint, in conventional terms, is "an act or short spell of running at full speed over a short distance". For New Atoms, a sprint, similarly, indicates a short duration of steady working that is fixed in advance (at this point, around two days).
+
+At the beginning of a sprint, the team votes on which [achievables](../glossary/achievable.md) will be put into "Next" on the [achievables board](https://github.com/newatoms/guides/tree/ready/board-guide). During a sprint, each team member works consistently on achievables (also called "cards") from the *Next* column of the achievables board and reviews the achievables fulfilled by other team members. At the end of a sprint, there is a demo during which we explain what we've done and why.
```
Comment (SPrinss) [inline glossary/sprint.md]: "... on achievables (also called "cards") .. "
I would remove cards (bit confusing) and maybe emphasize that we only work on the voted in achievables ?

"demo" should actually have it's own glossary probably but that might not be a priority now :)

**Comment by krystenkoehn** [comment]
Helpful, thanks! I'll make a card for "demo".

**Pull request merged**